src/Traits/DetectsChanges.php

```diff
@@ -154,4 +157,13 @@ protected static function getRelatedModelAttributeValue(Model $model, string $at
 
         return ["{$relatedModelName}.{$relatedAttribute}" => $relatedModel->$relatedAttribute ?? null];
     }
+
+    protected static function jsonLogAttribute(Model $model, string $attribute)
+    {
+        $path = preg_split('/(->)/', $attribute);
+        $modelAttribute = array_shift($path);
+        $modelAttribute = collect($model->getAttribute($modelAttribute));
+
+        return Arr::get($modelAttribute->toArray() ?? [], implode('.', $path));
```

Comment:
Isn't the `Collection::get()` method also dot-notation compatible? 🤔

Reply:
The `Collection::get()` wasn't playing nice, but I will try it again. It would look cleaner and remove an extra `use`;

Reply:
`data_get()` could also be an option to prevent the `toArray()` call in our code base.^^